deploy/stacks/monitoring.py

```diff
@@ -131,11 +137,19 @@ def create_cw_alarms(
         self.set_aurora_alarms(
             f'{resource_prefix}-{envname}-aurora-alarm', database, self.cw_alarm_action
         )
-        self.set_es_alarms(
-            f'{resource_prefix}-{envname}-opensearch-alarm',
-            openseach_domain,
-            self.cw_alarm_action,
-        )
+        if openseach_domain:
```

Comment:
What about when there is no opensearch_domain? no alarm?

Reply:
If there is no `opensearch_domain`, there should be `opensearch_serverless_collection_id` that has separate monitoring, including alarms on availability of the collection (collections are domains in serverless). The reason I separated the two is because serverless provides a different set of metrics. 

I can either put serverless monitoring into else clause here or raise an error if none of these two parameters are provided - whichever works best for you?

Reply:
Updated